diesel_tests/tests/expressions/mod.rs

```diff
@@ -48,7 +48,27 @@ table! {
 }
 
 #[test]
-fn test_count_max() {
+fn test_min() {
+    use self::numbers::columns::*;
+    use self::numbers::table as numbers;
+
+    let connection = connection();
+    connection
+        .execute("INSERT INTO numbers (n) VALUES (2), (1), (5)")
+        .unwrap();
+    let source = numbers.select(min(n));
+
+    assert_eq!(Ok(Some(1)), source.first(&connection));
+    connection
+        .execute("DELETE FROM numbers WHERE n = 1")
+        .unwrap();
+    assert_eq!(Ok(Some(2)), source.first(&connection));
+    connection.execute("DELETE FROM numbers").unwrap();
+    assert_eq!(Ok(None::<i32>), source.first(&connection));
+}
```

Comment:
Sorry I just realized that all this code was moved from another part in the file. Can we leave this code where it was before? This just adds unnecessary git churn.

Reply:
Sounds good